iceoryx_posh/test/fuzztests/fuzz_helper.hpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2021 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef IOX_POSH_FUZZTESTS_FUZZHELPER_HPP
+#define IOX_POSH_FUZZTESTS_FUZZHELPER_HPP
+
+#include "roudi_fuzz.hpp"
+#include <memory>
+
+/// @brief Class to implement some help methods for the fuzz wrapper
+class FuzzHelper
```

Comment:
Since this class has no state at all wouldn't it make more sense to make all methods free functions and get rid of this class completely?

Reply:
I'm not sure if it makes more sense. Since it is only a test and will not a productive code, my suggestions would be to let it as a class.